WordPress/wporg-main-2022
# Add About subpages

Add redesigned subpages of w.org/about: Requirements, Features, Security, Roadmap, and History.

Closes #149 

### Screenshots

| Requirements | Features | Security | Roadmap | History |
|--------|-------|-------|-------|-------|
| ![localhost_8888_about-2_requirements-2_(Desktop)](https://user-images.githubusercontent.com/1017872/214740123-b515764b-0195-45fd-8ba9-28584596d615.png) | ![localhost_8888_about-2_features-2_(Desktop)](https://user-images.githubusercontent.com/1017872/214740334-b0a0576f-ca8d-40ec-bfbd-b046172a2a5a.png) | ![localhost_8888_about-2_security-2_(Desktop)](https://user-images.githubusercontent.com/1017872/214740408-a75122cc-93ec-4413-94a7-222c12e4b9a5.png) | ![localhost_8888_about-2_roadmap-2_(Desktop)](https://user-images.githubusercontent.com/1017872/214740609-b70a99c9-9217-4b8f-8b7f-891106ba0e69.png) | ![localhost_8888_about-2_history-2_(Desktop)](https://user-images.githubusercontent.com/1017872/214740636-f757191a-2e2d-4052-b6a5-372ee6ccd357.png) |

### How to test the changes in this Pull Request:

1. Add the following pages to your site:
```
'/about-2/',
'/about-2/requirements-2/',
'/about-2/features-2/',
'/about-2/security-2/',
'/about-2/roadmap-2/',
'/about-2/history-2/',
```
2. Browse the pages using the navigation and internal links on '/about-2/security-2/',
3. Check responsive layout
4. Check escaping of text in patterns
